osu.Game/Modes/Objects/Drawables/DrawableHitObject.cs

```diff
@@ -85,6 +84,19 @@ protected override void LoadComplete()
             Expire(true);
         }
 
+        private List<DrawableHitObject> nestedHitObjects;

```

Comment:
This seems redundant. Why not make it `Container<DrawableHitObject>` and let it be in the draw hierarchy? Then you don't need to add slider ticks to 2 lists.